### PR TITLE
Eric/symco 79 Allow assignment settings to affect options in a conversation

### PR DESCRIPTION
**Pull request by ericharper1**
To test:

- Run migrations
- Set options for assignment in admin like recording attempts, allow typed response, and allow self rating. Recording attempts defaults at 1, instead of specifying a 'retry' number like it does in the ticket -- was less confusing this way

Note: I completely refactored the code for completing a response. I personally feel it's cleaner and more stable. Let me know what you think.

### CODE REVIEW

File: simcon_project/conversation_templates/models/template_node.py [hunk 1]
```diff
@@ -25,6 +25,3 @@ class TemplateNode(models.Model):
 
     def __str__(self):
         return f"{self.parent_template}: {self.description}"
-
-    def get_absolute_url(self):
```
Comment (rosasck) [inline simcon_project/conversation_templates/models/template_node.py:29]: i like that we dont have this

**Review by devin-w-hill — APPROVED** [review]

**Review by Likhovodov — COMMENTED** [review]
Looks good. I have one suggestion:

Though it isn't necessary, I'd prefer if the choices were displayed on the same page as the recording (like they were in the past). This breaks up the journey into distinct different steps that are each signified by a page that the user sees. Being brought to a new page to make the decision breaks up the 'flow' of the conversation if you will. They would still have to click some kind of button like 'done recording' that would make the choices available and disable the record button if they still have recording attempts left.

**Comment by rosasck** [comment]
Screen Shot 2021-02-21 at 6 51 49 PM

**Comment by rosasck** [comment]
ignore me

**Review by rosasck — APPROVED** [review]

**Pull request merged**